docker/production.dockerfile

```diff
@@ -0,0 +1,17 @@
+# vim: ft=dockerfile:
+
+FROM usgsastro/miniconda3
+
+RUN conda update conda && \
+    conda install python=3.6 && \
+    conda config --add channels conda-forge && \
+    conda config --add channels usgs-astrogeology
+
+RUN conda install -c usgs-astrogeology isis
+
+RUN apt-get update && \
+    apt-get install libgl1 -y && \
+    rm -rf /var/lib/apt/lists/*
+
+RUN bash -lc 'python "$CONDA_PREFIX/scripts/isisVarInit.py"'
```

Comment:
I am concerned about this part. The data area is not installed as part of the anaconda install process. The data area is also quite large and I'd be worried about downloading it inside of the docker container. So, how would users get the container using a local copy of the data area?

Reply:
Probably something that should be added to docs he added, but `isisVarInit.py` should just set the variables and you would download on the host machine (not in the container) and do a `-v $ISISTESTDATA:/opt/conda/testData` or similar to mount your downloads into the container.

Reply:
@evindunn Can you add this to the installation via docker instructions?

Reply:
@jessemapel  @Kelvinrr how's [this](https://github.com/evindunn/ISIS3/tree/docker-image#installation-with-docker) ?